conftest.py

```diff
@@ -22,7 +22,8 @@ def _spark_builder():
     return SparkSession.builder \
         .master("local[2]") \
         .config("spark.hadoop.io.compression.codecs", "io.projectglow.sql.util.BGZFCodec") \
-        .config("spark.ui.enabled", "false")
+        .config("spark.ui.enabled", "false") \
+        .config("spark.sql.execution.arrow.pyspark.enabled", "true")
```

Comment:
Most environments (including Databricks) should have this enabled by now. It allows us to avoid converting numpy arrays to python lists when converting Spark <=> Pandas DataFrames.

Reply:
Our unit tests for reshaping are pretty short. I don't think it would affect test time elsewhere.